camel-k-maven-plugin/src/main/java/org/apache/camel/k/tooling/maven/processors/CatalogProcessor_2_x.java

```diff
@@ -76,7 +76,8 @@ public int getOrder() {
 
     @Override
     public boolean accepts(CamelCatalog catalog) {
-        return new Semver(catalog.getCatalogVersion(), Semver.SemverType.IVY).satisfies("[2.18,3]");
+        Semver semver = new Semver(catalog.getCatalogVersion(), Semver.SemverType.IVY);
+        return semver.isGreaterThanOrEqualTo("2.18.0") && semver.isLowerThan("4.0.0");
```

Comment:
3.0.0 ?

Reply:
is LowerThan not LowerOrEqual, check the tests.